src/keywords/geometrylist.cpp

```diff
@@ -5,8 +5,8 @@
 #include "base/lineparser.h"
 #include "classes/coredata.h"
 
-GeometryListKeyword::GeometryListKeyword::GeometryListKeyword(List<Geometry> &data, Geometry::GeometryType type)
-    : KeywordData<List<Geometry> &>(KeywordBase::GeometryListData, data), type_(type)
+GeometryListKeyword::GeometryListKeyword::GeometryListKeyword(std::vector<Geometry> &data, Geometry::GeometryType type)
+    : KeywordData<std::vector<Geometry> &>(KeywordBase::GeometryListData, data), type_(type)
```

Comment:
```suggestion
GeometryListKeyword::GeometryListKeyword::GeometryListKeyword(std::vector<Geometry> data, Geometry::GeometryType type)
    : KeywordData<std::vector<Geometry>>(KeywordBase::GeometryListData, data), type_(type)
```
In line with the trend happening for other keywords, rather than storing references to target objects as the keyword data (as is the case here) I have been moving old and writing new keywords to store the object itself in the keyword.  I used to think that storing a reference to the target list / vector was a cool idea, but it creates a an unnecessary separation between the keyword and the data it is supposed to be managing.